apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -0,0 +1,739 @@
+import { onDocumentReady, templateFromAttributes } from '@prairielearn/browser-utils';
+import { escapeHtml, html } from '@prairielearn/html';
+
+import { AssessmentInstanceRow } from '../../src/pages/instructorAssessmentInstances/instructorAssessmentInstances.types.js';
+
+declare global {
+  interface Window {
+    popoverSubmitViaAjax: (e: any, popover: JQuery) => void;
+  }
+}
+
+onDocumentReady(() => {
+  const dataset = document.getElementById('usersTable')?.dataset || null;
```

Comment:
Nit, the `|| null` should be unnecessary:

```suggestion
  const dataset = document.getElementById('usersTable')?.dataset;
```

Reply:
Removed in [0b3aa97](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/0b3aa97bfea69bfe855c4af0ac526d99182d6f6b).